lib/core/src/animation/animator/continuous.rs

```diff
@@ -84,24 +86,28 @@ impl ContinuousAnimatorData {
 // === ContinuousAnimator ===
 // ==========================
 
-/// This structure runs an animation with continuous time as its input.
+/// This structure runs an animation with continuous time in milliseconds.
```

Comment:
I think this doc could explain more. I don't know what is the meaning "the structure runs an animation". This docs should explain that animation is a registered event_loop callback calling repeatedly some animation_callback.

Reply:
I think this should be clear enough:
```rust
/// `ContinuousAnimator` registers itself in `EventLoop`, repeatedly calling an
/// `AnimationCallback` with the playback time in millisecond as its input.
```